src/containers/App/App.js

```diff
@@ -184,10 +186,17 @@ export function App({
   });
 
   const showLoadingState = isPropertiesPlaceholder || isMessagesPlaceholder;
-  const isFetchingConfig = isFetchingProperties || isFetchingMessages;
-  const loadingConfigError = propertiesError || messagesError;
+  const isFetchingConfig =
+    isFetchingProperties || isFetchingMessages || isFetchingExtensions;
```

Comment:
no need to include extensions in this, it's only for holding off on setting global config such as the selected namespace which may depend on deployment settings (e.g. single namespace mode)